## 86% of end-end tests have passed
```
Total executed tests: 78
Failed tests: 8
Expected to fail tests: 3
Passed tests: 67
```
```
IDs of failed tests: 702731,741054,741966,741968,740220,740221,702732,740222 
```
```
IDs of expected to fail tests: 702844,741924,703503 
```
<h3>Failed tests (8)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/hooks-screen-params&PR_ID=22446&APK_NAME=22446.apk&TR_CASE_IDS=702731,741054,741966,741968,740220,740221,702732,740222">Rerun failed tests</a></li><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831543">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><p><blockquote><div>Device 1: Looking for a pinned message by text: Message 4 </div><div>Device 1: Find `Button` by `xpath`: `//*[@content-desc='pinned-messages-menu']//*[starts-with(@text,'Message 4')]` </div></blockquote></p>

```
private_chats/test_one_to_one_chat.py:267: in test_1_1_chat_pin_messages
    pinned_message.long_press_without_release()
../views/base_element.py:330: in long_press_without_release
    action.click_and_hold(self.find_element()).perform()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by xpath: `//*[@content-desc='pinned-messages-menu']//*[starts-with(@text,'Message 4')]` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESaMaB8BuSFqbSqM6I">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESVLH1AcV4jmgAObfR">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestProfileOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831610">test_profile_back_up_seed_phrase_validation</a>, id: 741966 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div>Device 1: Find `Button` by `accessibility id`: `Okay, continue` </div></blockquote></p>

```
profile/test_profile.py:29: in test_profile_back_up_seed_phrase_validation
    self.profile.ok_continue_button.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `Okay, continue` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESVnxxnsulYt5uNYuZ">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestProfileMultipleDevices:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831612">test_profile_change_username</a>, id: 741968 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: LogInButton </div><div>## Signed in successfully!</div></blockquote></p>

```
profile/test_profile.py:178: in test_profile_change_username
    self.errors.verify_no_errors()
base_test_case.py:205: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 2: Updated username 'user 1' is not shown in the community channel
E    Device 2: Updated username is not shown in the mentions list
E    Device 2: Updated username is absent in the contacts list
E    Device 2: Can't find chat with updated username
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESZ0H72BvQp5ZPq6PP">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESPhKGQ9SlrA2NDhLS">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestFallbackMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831597">test_fallback_add_key_pair</a>, id: 741054 </b></td></tr><tr><td>

```
Test setup failed: profile/test_fallback.py:30: in prepare_devices
    self.recovery_phrase, self.public_key_3 = self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:38: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/profile_view.py:225: in backup_recovery_phrase
    self.ok_continue_button.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `Okay, continue` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831593">test_fallback_sync_with_error</a>, id: 740220 </b></td></tr><tr><td><p><blockquote><div>Device 3: Tap on found: Text </div><div>Device 3: Click system back button </div></blockquote></p>

```
Test setup failed: profile/test_fallback.py:30: in prepare_devices
    self.recovery_phrase, self.public_key_3 = self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:38: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/profile_view.py:225: in backup_recovery_phrase
    self.ok_continue_button.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `Okay, continue` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES8PrL6x8MQDrzKhbQ">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESEQ0TQqnLTyTDfYRv">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESXYLLWLLdI9piuID8">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831594">test_fallback_with_correct_seed_phrase</a>, id: 740221 </b></td></tr><tr><td>

```
Test setup failed: profile/test_fallback.py:30: in prepare_devices
    self.recovery_phrase, self.public_key_3 = self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:38: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/profile_view.py:225: in backup_recovery_phrase
    self.ok_continue_button.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `Okay, continue` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831595">test_fallback_validate_seed_phrase</a>, id: 740222 </b></td></tr><tr><td>

```
Test setup failed: profile/test_fallback.py:30: in prepare_devices
    self.recovery_phrase, self.public_key_3 = self.loop.run_until_complete(
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:38: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
../views/profile_view.py:225: in backup_recovery_phrase
    self.ok_continue_button.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `Okay, continue` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831544">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><p><blockquote><div>Device 2: `Text` is `shwiNYUhsC2mXqEx8YQR` </div><div>Device 2: Looking for a pinned by message with text: Message 4 </div></blockquote></p>

```
private_chats/test_group_chat.py:449: in test_group_chat_pin_messages
    self.errors.verify_no_errors()
base_test_case.py:205: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 2: Pinned messages count 2 doesn't match expected 3 for user shoda6gxkB4zQSVMN5aD
E    Device 1: Message 'Message 3' is missed on Pinned messages list for user shwiNYUhsC2mXqEx8YQR
E    Device 2: Message 'Message 3' is missed on Pinned messages list for user shoda6gxkB4zQSVMN5aD
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES2gS5Sif5ZXjjmv26">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESyrz2VHJ2SfOD4bRn">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESO6mHtpeKikiuArUK">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Expected to fail tests (3)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831563">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `EmojisNumber` by `xpath`: `//*[starts-with(@text,'https://m.youtube.com/watch?v=Je7yErjEVt4')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']/../..//*[@content-desc='emoji-reaction-4']/android.widget.TextView[2]` </div><div>Device 2: Element EmojisNumber text is equal to 1 </div></blockquote></p>

```
communities/test_community.py:665: in test_community_links_with_previews_github_youtube_twitter_gif_send_enable
    self.errors.verify_no_errors()
base_test_case.py:205: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 1: No preview image is shown for https://github.com/status-im/status-mobile/pull/11707
E    Device 1: No preview is loaded for url https://youtu.be/Je7yErjEVt4
E    Device 1: No preview is loaded for url https://www.youtube.com/watch?v=XN-SVmuJH2g&list=PLbrz7IuP1hrgNtYe9g6YHwHO6F3OqNMao
E    Device 1: No preview is loaded for url https://m.youtube.com/watch?v=Je7yErjEVt4 
```

<b>[[Youtube links preview is not loaded on LambdaTest emulators, needs investigation]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEShL1pG6jkxPUvofTc">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESmudsWxqREVumfgNE">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedThree:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831607">test_community_mobile_data_and_wi_fi_only_fetching</a>, id: 741924 </b></td></tr><tr><td><p><blockquote><div>Device 2: Click until `ChatMessageInput` by `accessibility id`: `chat-message-input` will be presented </div><div>Device 2: Looking for a message by text: message text 2 </div></blockquote></p>

```
communities/test_community.py:1359: in test_community_mobile_data_and_wi_fi_only_fetching
    self.errors.verify_no_errors()
base_test_case.py:205: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 2: Can't receive the message 'message text 1' in community when using mobile data
E    Device 2: Message 'message text 2' in community, which is sent when receiver was offline, is received using mobile data 
```

<b>[[Community message can be fetched from offline when using mobile data]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESj77xRngJDxrlMrhI">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES2OwmTf4C84aibpJ6">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831585">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><p><blockquote><div>Device 1: Image differs from template to 0.0 percents </div><div>Device 1: Find `BaseElement` by `xpath`: `//*[@content-desc='community-title']/preceding-sibling::*/android.widget.ImageView` </div></blockquote></p>

```
communities/test_community.py:294: in test_community_discovery
    if self.community_view.community_logo.is_element_differs_from_template('status_community_logo.png'):
../views/base_element.py:267: in is_element_differs_from_template
    difference = ImageChops.difference(self.image, self.template)
../views/base_element.py:245: in image
    return Image.open(BytesIO(base64.b64decode(self.find_element().screenshot_as_base64)))
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: BaseElement by xpath: `//*[@content-desc='community-title']/preceding-sibling::*/android.widget.ImageView` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception 
```

<b>[[Might fail if the Discover curation dapp is too slow to load > 2 mins]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES5yrWCum1PMt027OF">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (67)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831568">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESaMaB8BuSFqbSqM6I">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESVLH1AcV4jmgAObfR">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831542">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESaMaB8BuSFqbSqM6I">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESVLH1AcV4jmgAObfR">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831547">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESaMaB8BuSFqbSqM6I">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESVLH1AcV4jmgAObfR">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831545">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESaMaB8BuSFqbSqM6I">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESVLH1AcV4jmgAObfR">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831557">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESaMaB8BuSFqbSqM6I">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESVLH1AcV4jmgAObfR">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831550">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESaMaB8BuSFqbSqM6I">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESVLH1AcV4jmgAObfR">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831582">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESaMaB8BuSFqbSqM6I">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESVLH1AcV4jmgAObfR">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestProfileOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831611">test_profile_change_currency</a>, id: 741967 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESVnxxnsulYt5uNYuZ">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831564">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESgSzhxYhgWfv4TP1X">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESaNgPch5R2lqxhQAO">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831553">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESgSzhxYhgWfv4TP1X">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESaNgPch5R2lqxhQAO">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831556">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESgSzhxYhgWfv4TP1X">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESaNgPch5R2lqxhQAO">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831573">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESgSzhxYhgWfv4TP1X">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESaNgPch5R2lqxhQAO">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831586">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESgSzhxYhgWfv4TP1X">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESaNgPch5R2lqxhQAO">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831575">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESxVyLBSc6pMAxDRBm">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESwBMYAcq4VsAajv6R">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831574">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESxVyLBSc6pMAxDRBm">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESwBMYAcq4VsAajv6R">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831603">test_activity_centre_contact_request_receiver_offline</a>, id: 741809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESxVyLBSc6pMAxDRBm">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESwBMYAcq4VsAajv6R">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831618">test_wallet_send_erc20_from_drawer[Sepolia-USD Coin-USDC-2-0.01]</a>, id: 742016 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESchwUcD4jtyqsQoUj">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESiyy3odCpCEa38QkC">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831617">test_send_eth[Arbitrum Sepolia-0.0001]</a>, id: 742015 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESchwUcD4jtyqsQoUj">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESiyy3odCpCEa38QkC">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831589">test_send_eth[Status Network Sepolia-0.0002]</a>, id: 727229 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESchwUcD4jtyqsQoUj">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESiyy3odCpCEa38QkC">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831590">test_wallet_send_erc20_from_drawer[Optimism Sepolia-USD Coin-USDC-2-0.01]</a>, id: 727230 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESchwUcD4jtyqsQoUj">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESiyy3odCpCEa38QkC">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831572">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESvxnjHBrGiHYsPOgb">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESo8tNGkwFRq9BvcBI">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestProfileMultipleDevices:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831615">test_profile_change_accent_color</a>, id: 741971 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESZ0H72BvQp5ZPq6PP">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESPhKGQ9SlrA2NDhLS">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831613">test_profile_change_profile_photo</a>, id: 741969 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESZ0H72BvQp5ZPq6PP">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESPhKGQ9SlrA2NDhLS">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831616">test_profile_allow_new_contact_requests_toggle</a>, id: 741972 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESZ0H72BvQp5ZPq6PP">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESPhKGQ9SlrA2NDhLS">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831614">test_profile_set_bio</a>, id: 741970 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESZ0H72BvQp5ZPq6PP">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESPhKGQ9SlrA2NDhLS">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDeviceThree:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831619">test_wallet_swap_dai_snt_real_tx</a>, id: 742063 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESjIDkqcRgu2QUhkBe">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831579">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES2gS5Sif5ZXjjmv26">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESyrz2VHJ2SfOD4bRn">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESO6mHtpeKikiuArUK">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831554">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES2gS5Sif5ZXjjmv26">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESyrz2VHJ2SfOD4bRn">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESO6mHtpeKikiuArUK">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831555">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES2gS5Sif5ZXjjmv26">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESyrz2VHJ2SfOD4bRn">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESO6mHtpeKikiuArUK">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831580">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES2gS5Sif5ZXjjmv26">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESyrz2VHJ2SfOD4bRn">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESO6mHtpeKikiuArUK">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831583">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES2gS5Sif5ZXjjmv26">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESyrz2VHJ2SfOD4bRn">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESO6mHtpeKikiuArUK">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831587">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESzb6HJFKDvU4RIkGn">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831548">test_links_deep_links_profile</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESzb6HJFKDvU4RIkGn">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831592">test_deep_links_communities</a>, id: 739307 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESzb6HJFKDvU4RIkGn">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831560">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEShL1pG6jkxPUvofTc">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESmudsWxqREVumfgNE">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831571">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEShL1pG6jkxPUvofTc">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESmudsWxqREVumfgNE">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831576">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEShL1pG6jkxPUvofTc">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESmudsWxqREVumfgNE">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831561">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEShL1pG6jkxPUvofTc">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESmudsWxqREVumfgNE">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831559">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEShL1pG6jkxPUvofTc">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESmudsWxqREVumfgNE">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831558">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEShL1pG6jkxPUvofTc">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESmudsWxqREVumfgNE">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831588">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEShL1pG6jkxPUvofTc">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESmudsWxqREVumfgNE">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831562">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEShL1pG6jkxPUvofTc">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESmudsWxqREVumfgNE">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831569">test_community_one_image_send_reply_set_reaction</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEShL1pG6jkxPUvofTc">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESmudsWxqREVumfgNE">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831578">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEShL1pG6jkxPUvofTc">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESmudsWxqREVumfgNE">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestAndroid13:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831602">test_create_account_android_13</a>, id: 741807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESxwrYI73MQGfBYesv">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestAndroid12:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831601">test_create_account_android_12</a>, id: 741806 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES2H2ClwNXdXKkPEdJ">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831584">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESHAfcOwAxB66ajX5D">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESYd7NABKP5spf3o3M">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831551">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESHAfcOwAxB66ajX5D">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESYd7NABKP5spf3o3M">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831552">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESHAfcOwAxB66ajX5D">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESYd7NABKP5spf3o3M">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletCollectibles:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831605">test_wallet_send_collectible</a>, id: 741840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESQNTxYshAEFZy3hYs">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831604">test_wallet_collectibles_balance</a>, id: 741839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESQNTxYshAEFZy3hYs">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831606">test_wallet_collectible_send_from_expanded_info_view</a>, id: 741841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESQNTxYshAEFZy3hYs">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDeviceTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831591">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESCYHD2Kz5TEgLVMBn">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831577">test_restore_multiaccount_with_waku_backup_remove_profile_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES5yrWCum1PMt027OF">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831546">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES5yrWCum1PMt027OF">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831565">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES5yrWCum1PMt027OF">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831570">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES5yrWCum1PMt027OF">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831581">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES5yrWCum1PMt027OF">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831599">test_wallet_swap_flow_mainnet</a>, id: 741555 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESuBJINWrJLFKCNNsz">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831596">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESuBJINWrJLFKCNNsz">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831600">test_wallet_bridge_flow_mainnet</a>, id: 741612 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESuBJINWrJLFKCNNsz">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831598">test_wallet_send_flow_mainnet</a>, id: 741554 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESuBJINWrJLFKCNNsz">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedThree:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831608">test_community_send_message_from_offline</a>, id: 741925 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESj77xRngJDxrlMrhI">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES2OwmTf4C84aibpJ6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831609">test_community_messaging_on_mobile_data</a>, id: 741926 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESj77xRngJDxrlMrhI">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES2OwmTf4C84aibpJ6">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831567">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESG7IGUhoRktAFJxyK">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESH5GXjGSH5dppFlYA">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831566">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESG7IGUhoRktAFJxyK">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESH5GXjGSH5dppFlYA">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3831549">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESG7IGUhoRktAFJxyK">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESH5GXjGSH5dppFlYA">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/legacy/status_im/ui/screens/communities/members.cljs
+++ b/src/legacy/status_im/ui/screens/communities/members.cljs
@@ -111,32 +111,31 @@
 
 (defn view
   []
-  (let [{:keys [community-id]} (quo.context/use-screen-params)]
+  (let [{:keys [community-id]} (quo.context/use-screen-params)
+        my-public-key (rf/sub [:multiaccount/public-key])
+        {:keys [permissions
+                can-manage-users?
+                admin]}
+        (rf/sub [:communities/community community-id])
+        sorted-members (rf/sub [:communities/sorted-community-members
+                                community-id])]
     (rn/use-mount
      #(rf/dispatch [:community/fetch-requests-to-join community-id]))
-    (fn []
-      (let [my-public-key (rf/sub [:multiaccount/public-key])
-            {:keys [permissions
-                    can-manage-users?
-                    admin]}
-            (rf/sub [:communities/community community-id])
-            sorted-members (rf/sub [:communities/sorted-community-members
-                                    community-id])]
-        [:<>
-         [topbar/topbar
-          {:title    (i18n/label :t/community-members-title)
-           :subtitle (str (count sorted-members))}]
-         [header community-id]
-         (when (and can-manage-users? (= constants/community-on-request-access (:access permissions)))
-           [requests-to-join community-id])
-         [rn/flat-list
-          {:data        sorted-members
-           :render-data {:community-id      community-id
-                         :my-public-key     my-public-key
-                         :can-kick-users?   (and can-manage-users?
-                                                 (not= (:access permissions)
-                                                       constants/community-no-membership-access))
-                         :can-manage-users? can-manage-users?
-                         :admin?            admin}
-           :key-fn      identity
-           :render-fn   render-member}]]))))
+    [:<>
+     [topbar/topbar
+      {:title    (i18n/label :t/community-members-title)
+       :subtitle (str (count sorted-members))}]
+     [header community-id]
+     (when (and can-manage-users? (= constants/community-on-request-access (:access permissions)))
+       [requests-to-join community-id])
+     [rn/flat-list
+      {:data        sorted-members
+       :render-data {:community-id      community-id
+                     :my-public-key     my-public-key
+                     :can-kick-users?   (and can-manage-users?
+                                             (not= (:access permissions)
+                                                   constants/community-no-membership-access))
+                     :can-manage-users? can-manage-users?
+                     :admin?            admin}
+       :key-fn      identity
+       :render-fn   render-member}]]))

--- a/src/status_im/common/emoji_picker/view.cljs
+++ b/src/status_im/common/emoji_picker/view.cljs
@@ -170,8 +170,7 @@
 
 (defn view
   [_]
-  (let [{:keys [on-select]}       (quo.context/use-screen-params)
-        scroll-ref                (atom nil)
+  (let [scroll-ref                (atom nil)
         set-scroll-ref            #(reset! scroll-ref %)
         search-text               (reagent/atom "")
         filtered-data             (reagent/atom nil)
@@ -197,14 +196,15 @@
                                       :active-category                active-category
                                       :should-update-active-category? (nil? @filtered-data)}))]
     (fn [sheet-opts]
-      [sheet-view
-       (assoc sheet-opts
-              :search-active?            (pos? (count @search-text))
-              :on-change-text            on-change-text
-              :clear-states              clear-states
-              :filtered-data             @filtered-data
-              :set-scroll-ref            set-scroll-ref
-              :on-select                 on-select
-              :on-viewable-items-changed on-viewable-items-changed
-              :active-category           active-category
-              :scroll-ref                scroll-ref)])))
+      (let [{:keys [on-select]} (quo.context/use-screen-params)]
+        [sheet-view
+         (assoc sheet-opts
+                :search-active?            (pos? (count @search-text))
+                :on-change-text            on-change-text
+                :clear-states              clear-states
+                :filtered-data             @filtered-data
+                :set-scroll-ref            set-scroll-ref
+                :on-select                 on-select
+                :on-viewable-items-changed on-viewable-items-changed
+                :active-category           active-category
+                :scroll-ref                scroll-ref)]))))

--- a/src/status_im/common/lightbox/view.cljs
+++ b/src/status_im/common/lightbox/view.cljs
@@ -138,17 +138,11 @@
          :transparent?          transparent?
          :bottom-text-component bottom-text-component}])]))
 
-(defn- f-lightbox
-  []
-  (let [{:keys [images index bottom-text-component on-options-press]}
-        (quo.context/use-screen-params)
-        props
-        (utils/init-props)
-        state
-        (utils/init-state images index)
-        handle-items-changed
-        (fn [e]
-          (on-viewable-items-changed e props state))]
+(defn- lightbox-internal
+  [{:keys [images index bottom-text-component on-options-press]}]
+  (let [props                (utils/init-props)
+        state                (utils/init-state images index)
+        handle-items-changed (fn [e] (on-viewable-items-changed e props state))]
     (fn []
       (let [animations (utils/init-animations (count images) index)
             derived    (utils/init-derived-animations animations)]
@@ -170,4 +164,5 @@
 
 (defn lightbox
   []
-  [:f> f-lightbox])
+  (let [screen-params (quo.context/use-screen-params)]
+    [lightbox-internal screen-params]))

--- a/src/status_im/contexts/profile/edit/modal/view.cljs
+++ b/src/status_im/contexts/profile/edit/modal/view.cljs
@@ -187,10 +187,9 @@
         (and new-picture? acceptable-name?)
         (and new-name? valid-name?))))
 
-(defn view
-  []
-  (let [{:keys [pending-event]}        (quo.context/use-screen-params)
-        {initial-display-name :display-name
+(defn view-internal
+  [{:keys [pending-event]}]
+  (let [{initial-display-name :display-name
          initial-color        :customization-color
          [initial-image]      :images} (rf/sub [:profile/profile])
         top                            (safe-area/get-top)
@@ -265,3 +264,8 @@
             :scroll-y     @scroll-y
             :on-submit    update-profile
             :disabled?    disabled?}]]]))))
+
+(defn view
+  []
+  (let [screen-params (quo.context/use-screen-params)]
+    [view-internal screen-params]))

--- a/src/status_im/contexts/wallet/add_account/add_address_to_watch/confirm_address/view.cljs
+++ b/src/status_im/contexts/wallet/add_account/add_address_to_watch/confirm_address/view.cljs
@@ -16,15 +16,15 @@
 
 (defn view
   []
-  (let [{:keys [address]} (quo.context/use-screen-params)
-        placeholder       (i18n/label :t/default-watched-address-placeholder)
+  (let [placeholder       (i18n/label :t/default-watched-address-placeholder)
         account-name      (reagent/atom "")
         account-color     (reagent/atom (rand-nth colors/account-colors))
         account-emoji     (reagent/atom (emoji-picker.utils/random-emoji))
         name-error        (reagent/atom nil)
         emoji-color-error (reagent/atom nil)]
     (fn []
-      (let [accounts-names             (rf/sub [:wallet/accounts-names])
+      (let [{:keys [address]}          (quo.context/use-screen-params)
+            accounts-names             (rf/sub [:wallet/accounts-names])
             accounts-emojis-and-colors (rf/sub [:wallet/accounts-emojis-and-colors])
             on-change-name             (fn [new-name]
                                          (reset! account-name new-name)

--- a/src/status_im/contexts/wallet/add_account/create_account/new_keypair/confirm_backup/view.cljs
+++ b/src/status_im/contexts/wallet/add_account/create_account/new_keypair/confirm_backup/view.cljs
@@ -116,8 +116,8 @@
                                         (on-try-again)))}
        :blur?            shell?}]]))
 
-(defn view
-  []
+(defn- view-internal
+  [{:keys [on-success on-try-again masked-seed-phrase back-button? theme shell?]}]
   (let [random-indices  (random-selection)
         quiz-index      (reagent/atom 0)
         incorrect-count (reagent/atom 0)
@@ -128,12 +128,7 @@
        (fn []
          (native-module/get-random-mnemonic #(reset! random-phrase (string/split % #"\s")))))
       (when-not (empty? @random-phrase)
-        (let [{:keys [on-success
-                      on-try-again
-                      masked-seed-phrase
-                      back-button?
-                      theme shell?]}        (quo.context/use-screen-params)
-              unmasked-seed-phrase          (security/safe-unmask-data masked-seed-phrase)
+        (let [unmasked-seed-phrase          (security/safe-unmask-data masked-seed-phrase)
               current-word-index            (get random-indices
                                                  (min @quiz-index (dec questions-count)))
               current-word                  (get unmasked-seed-phrase current-word-index)
@@ -217,3 +212,8 @@
              {:on-press on-button-press
               :options  options-row-1
               :shell?   shell?}]]])))))
+
+(defn view
+  []
+  (let [screen-params (quo.context/use-screen-params)]
+    [view-internal screen-params]))

--- a/src/status_im/contexts/wallet/add_account/create_account/new_keypair/confirm_backup/view.cljs
+++ b/src/status_im/contexts/wallet/add_account/create_account/new_keypair/confirm_backup/view.cljs
@@ -118,23 +118,23 @@
 
 (defn view
   []
-  (let [random-indices         (random-selection)
-        quiz-index             (reagent/atom 0)
-        incorrect-count        (reagent/atom 0)
-        show-error?            (reagent/atom false)
-        {:keys [on-success
-                on-try-again
-                masked-seed-phrase
-                back-button?
-                theme shell?]} (quo.context/use-screen-params)
-        unmasked-seed-phrase   (security/safe-unmask-data masked-seed-phrase)
-        random-phrase          (reagent/atom [])]
+  (let [random-indices  (random-selection)
+        quiz-index      (reagent/atom 0)
+        incorrect-count (reagent/atom 0)
+        show-error?     (reagent/atom false)
+        random-phrase   (reagent/atom [])]
     (fn []
       (rn/use-mount
        (fn []
          (native-module/get-random-mnemonic #(reset! random-phrase (string/split % #"\s")))))
       (when-not (empty? @random-phrase)
-        (let [current-word-index            (get random-indices
+        (let [{:keys [on-success
+                      on-try-again
+                      masked-seed-phrase
+                      back-button?
+                      theme shell?]}        (quo.context/use-screen-params)
+              unmasked-seed-phrase          (security/safe-unmask-data masked-seed-phrase)
+              current-word-index            (get random-indices
                                                  (min @quiz-index (dec questions-count)))
               current-word                  (get unmasked-seed-phrase current-word-index)
               [options-row-0 options-row-1] (random-words-with-string @random-phrase current-word)

